src/test/java/hillelJavaEE_02/greetings/GreetingComponentTest.java

```diff
@@ -0,0 +1,40 @@
+package hillelJavaEE_02.greetings;
+
+import org.junit.Test;
+
+import java.util.*;
+
+import static java.lang.Math.pow;
+import static org.hamcrest.Matchers.lessThan;
+import static org.junit.Assert.assertThat;
+
+public class GreetingComponentTest {
+
+    @Test
+    public void getRandomGreeting() {
+        // math.semestr.ru/group/uniform-distribution.php
+
+        GreetingComponent gC = new GreetingComponent();
+        List<String> gList = gC.getGreetingsList();
+        Map<Integer, Integer> distribution = new HashMap<>();
+
+        for (int i = 0; i < 1_000_000; i++) {
+            Integer randomValue = gList.indexOf(gC.getRandomGreeting()) + 1;
+            if(distribution.get(randomValue) == null) {
+                distribution.put(randomValue, 1);
+            } else {
+                distribution.put(randomValue, distribution.get(randomValue) + 1);
+            }
+        }

```

Comment:
try to:
avoid using names like `gC` (GC is a garbage collector)
avoid using `List.indexOf()`
use Map.compute() instead of `if .. then put(1) else put(+1)`